test/scrabble_test.rb

```diff
@@ -5,9 +5,36 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  def test_can_read_values_of_each_letter
+    values = {
+      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+      "Y"=>4, "Z"=>10
+    }
+    assert_equal values, Scrabble.new.values
+  end
+
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_a_word
+    assert_equal 8, Scrabble.new.score('Hello')
+    assert_equal 14, Scrabble.new.score('Quiet')
+    assert_equal 6, Scrabble.new.score('Bed')
+    assert_equal 5, Scrabble.new.score('Deal')
+    assert_equal 12, Scrabble.new.score('Zoo')
+  end
+
+  def test_empty_or_nil_scores_0
+    assert_equal 0, Scrabble.new.score(nil)
+    assert_equal 0, Scrabble.new.score("")
+  end
```

Comment:
Like this test. Breaking out high level logic into smaller methods would allow you to include unit tests as well instead of only integration tests.